bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -234,7 +236,7 @@ public Injector initialize()
         }
 
         // Log effective configuration
-        if (!quiet) {
+        if (!((quietConfig == null) ? quiet : quietConfig)) {
```

Comment:
`Bootstrap.quiet()` (`quiet`) is local to invocation, so should take precedence over config file.

Reply:
Can we log a single message saying that this is effective? It'll help to figure out why the configuration is not being logged.

Reply:
No, the `quiet()` call is intended to act as a default for the configuration, so the configuration can override in either direction.

Reply:
@nineinchnick The intent of this flag is to not log anything, so logging a message saying we're not logging anything would be counterproductive.